render/render.js

```diff
@@ -64,12 +71,16 @@ module.exports = function($window) {
 		insertNode(parent, vnode.dom, nextSibling)
 		return vnode.dom
 	}
-	function createHTML(parent, vnode, nextSibling) {
+	var possibleParents = {caption: "table", thead: "table", tbody: "table", tfoot: "table", tr: "tbody", th: "tr", td: "tr", colgroup: "table", col: "colgroup"}
```

Comment:
I appreciate pulling this out. (I almost did over half a dozen times, and just didn't.)